bibliopixel/animation/sequence.py

```diff
@@ -47,4 +48,4 @@ def step(self, amt=1):
                 self.index = 0
 
         if not self.completed and self.animations:
-            self.current_animation.run_all_frames()
+            self.current_animation.start()
```

Comment:
I thought I made it so sequence animations are ALWAYS threaded so that I could kill them. run_all_frames means you cannot kill a sequence until an animation completes entirely. Using start() means it will die after the current frame.
This has gotta stay so that remote even works at all.